chichiwang/exercise-js
# Add section to Documentation page: Application Layouts

## Description
Linked to Issue: #113 
Add section to `/documentation` page "Application Layouts" that mirrors the content of `docs/README.md#application-layouts`. Additionally, fix incorrect hook names that appear in `app/hooks/README.md`.

## Changes
* Fix incorrect hook names in `app/hooks/README.md`
* Fix hash link to hook `useResetScroll` documentation in `docs/README.md`
* Create component `pages/Documentation/ApplicationLayouts.tsx`
* Render component `ApplicationLayouts` in `pages/Documentation/index.tsx`
* Add link to section "Application Layouts" in `pages/Documentation/index.tsx`

## Steps to QA
* Verify changes in the README files
  * `docs/README.md`
  * `app/hooks/README.md`
* Pull down and switch to this branch
* Run the app and verify in browser
  * The "Application Layouts" section of the `/documentation` page looks correct
  * The link to the "Application Layouts" section under the "Developing Locally" section works
  * The links in the "Application Layouts" section works
  * The copy button in the code snippet in the "Application Layouts" section works
  * Copying the code manually from the code snippet in the "Application Layouts" section works
